tests/modules/nf-core/mageck/test/nextflow.config

```diff
@@ -0,0 +1,12 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: MAGECK_TEST {
+        ext.args = "-t 'HL60.initial' "
+    }
+
+    withName: MAGECK_TEST_DAY0_LABEL {
```

Comment:
```suggestion
    withName: 'test_mageck_test_day0_label:MAGECK_TEST' {
```

Reply:
These params weren't having any effect because they weren't updated to match the workflow. My fault for suggesting the single module import!

I'm assuming the tests should have different file outputs in some way and they're currently identicial so the md5sums will need updating in the test.yml